infra/azure/terraform/infrastructure/main.tf

```diff
@@ -0,0 +1,74 @@
+# Data source to get the current client configuration
+data "azurerm_client_config" "current" {}
+
+# Resource group for CAPZ CI resources
+resource "azurerm_resource_group" "capz_ci" {
+  location = "eastus"
+  name     = "capz-ci"
+  tags = {
+    DO-NOT-DELETE = "contact capz"

```

Comment:
maybe we should update this to a sig-cluster-lifecycle-azure email (@jackfrancis do you have a email that should be used?)  and add the sig-infra/sig-test email as a contact?

Reply:
I don't know if we have an email to share, #cluster-api-provider-azure on k8s slack or the SIG mailing list are probably our best bet:

- https://groups.google.com/g/kubernetes-sig-cluster-lifecycle

Reply:
I don't think it would be on call, just a pointer to someone that would know what the Resource group does?  I we can just keep what is there now.